src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -16,19 +16,42 @@ redirects:
   - /docs/browser/new-relic-browser/installation/install-browser-monitoring-agent
 ---
 
-Browser uses a [JavaScript snippet](/docs/browser/new-relic-browser/page-load-timing/instrumentation-page-load-timing) (or "agent") to instrument your app's webpages. The JavaScript collects data for browser monitoring. To install the browser agent, you can choose from a number of deployment options.
+Browser uses a [JavaScript snippet](/docs/browser/new-relic-browser/page-load-timing/instrumentation-page-load-timing), also referred to as an "agent," to instrument your app's webpages. The JavaScript collects data for browser monitoring. To install the browser agent, you can choose from a number of deployment options.
```

Comment:
Shouldn't it be "agent", ?

Reply:
that's how most of the world does it, but U.S. is dumb in that regard. Personally I do it the way you do and the way the rest of the world does it.